src/test/components/CompareHome.test.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, fireEvent, cleanup } from 'react-testing-library';
+
+import CompareHome from '../../components/app/CompareHome';
+import { getProfilesToCompare } from '../../selectors/url-state';
+
+import { blankStore } from '../fixtures/stores';
+
+describe('app/CompareHome', () => {
+  afterEach(cleanup);
+
+  it('renders properly', () => {
+    const store = blankStore();
+    const { container, getByLabelText, getByText } = render(
+      <Provider store={store}>
+        <CompareHome />
+      </Provider>
+    );
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Please break this into multiple `it` blocks. I would suggest the first to be called `it('matches the snapshot')` and then something like `it('can compare profiles, given two URLs')`. This way a snapshot failure will be more obvious compared to a logic failure.